src/main/java/ro/itschool/ema/models/entities/User.java

```diff
@@ -0,0 +1,34 @@
+package ro.itschool.ema.models.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.MappedSuperclass;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+import lombok.*;
+import org.springframework.validation.annotation.Validated;
+
+import java.util.Date;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@MappedSuperclass
+@Validated
+public abstract class User {
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+    @Column(name = "date_of_birth")
+    private Date dateOfBirth;
+
+    @Column(name = "email")
```

Comment:
Maybe have something like this
@Column(name = "email", unique = true)
@Email(message = "Invalid email. Please enter a valid email.")

we need to make sure that the email is unique